.github/workflows/ci.yml

```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codestyle-check:
+    name: "Codestyle check"
+    runs-on: ubuntu-20.04
```

Comment:
This job should depend on the PHP linter. (If the PHP is invalid, it does not make sense to check for duplication.)